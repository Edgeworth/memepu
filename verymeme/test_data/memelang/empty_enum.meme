// status: ok
// input: ''
// output: ''
// returns: 0
enum Unit {}
fn main() i8 { ret 0; }
