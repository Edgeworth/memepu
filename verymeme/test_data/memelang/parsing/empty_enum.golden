File
 Enum(0 untyped)
  Typename(Unit)
